test/e2e/utils/remoteexec/command.go

```diff
@@ -0,0 +1,62 @@
+package remoteexec
```

Comment:
we can relicense as `kubernetes authors` (I wrote all of it) - let's add boilerplate license text here and for all the code in `test/e2e/utils`